src/mson_to_json_schema.js

```diff
@@ -1,9 +1,9 @@
 var escapeJSONPointer = require('./escape_json_pointer');
 
-function convertMsonToJsonSchema(content) {
+function convertMsonToJsonSchema(content, openApi3) {
```

Comment:
Instead of explicitly passing in `openapi3`, let's use the `options` variable or `context` to derive the state